### ISSUE TITLE
Keyboard Navigation

### ISSUE THREAD

**Issue by tommoor** [speak/issues#157]
Something I know i'd like for the team list, adding it here to see if other folks feel the same way as that'll decide priority for implementing :smile:

**Comment by fuzzbomb** [comment]
This should be classified as a BUG form an accessibility perspective.
I can't see any way to change the labels or issue title - can that only be done by the original reporter?


- For a mouse/trackpad user, keyboard controls could be regarded as an "enhancement".
- For a keyboard-only user (not capable of using a mouse easily) the people and teams menus are completely inaccessible, so it's a bug.
  - When I hold down ALT, the menu bar appears. File and Help have underlined letters (aka Menu Accelerators). I can access those menus (with ALT+F and ALT+H respectively) and use the up/down arrow keys to move within those menus.
  - People and Team do NOT have underlined letters, and I cannot access those menus without resorting to a mouse.
  - Left/right arrows do NOT allow you to move from one menu to the next. This is expected behaviour. Indeed, it would mitigate the previous problem; if left/right allowed a user to move from one menu to the next, then I could activate the File menu with ALT+F, then use right arrow to get into the People and Team menus.


I'm using KDE Plasma 5 on Ubuntu Linux 15.10.
